modules/Movecraft/src/main/java/net/countercraft/movecraft/async/AsyncManager.java

```diff
@@ -525,7 +525,7 @@ private void detectSinking(){
                     }
                 }
 
-                if (blockID != 0) {
+                if (blockID != 0 && blockID != 51) {
                     totalNonAirBlocks++;

```

Comment:
Change the variable name to be indicative of this change in purpose. I recommend `negligibleBlocks` as an option.

Reply:
@drfiveminusmint